keps/sig-cluster-lifecycle/kubeadm/0029-20180918-kubeadm-phases-beta.md

```diff
@@ -152,7 +156,8 @@ workflows e.g. reuse of phase `certs` in both `kubeadm init` and `kubeadm join`
 * [#61631](https://github.com/kubernetes/kubernetes/pull/61631) First prototype implementation 
   (now outdated)
 * v1.13 implementation of phases in the `kubeadm init` workflow
-* v1.14 implementation of phases in the `kubeadm join` workflow (planned)
+* v1.14 implementation of phases in the `kubeadm join` workflow 
+* v1.17 implementation of phases in the `kubeadm upgrade apply/plan/node` workflow (planned)
```

Comment:
[1]

Reply:
v1.17 is `kubeadm upgrade apply/plan`. `kubeadm upgrade node` was done in one of the previous cycles (although I can't remember which one).